problem1.js

```diff
@@ -0,0 +1,6 @@
+
+
+const sum = (...arg) => (arg.reduce((x,y)=>x+y))
+
```

Comment:
Should add some comments as a general good practice. Also, beware of the extra lines.